meerk40t/balormk/gui/balorconfig.py

```diff
@@ -97,13 +111,37 @@ def test_pin(self, value):
         else:
             self.context("red off\n")
 
+    def update_bit_info(self, *args):
```

Comment:
This entire thing belongs as a wait call in the controller. We are very likely waiting for a footpedal like event or something. There's some similar stuff in the controller/driver already.

Reply:
Ah, no. Okay I see what this is doing.